src/coreclr/build-runtime.cmd

```diff
@@ -455,7 +469,7 @@ if %__BuildCrossArchNative% EQU 1 (
         set __ExtraCmakeArgs="-DCMAKE_BUILD_TYPE=!__BuildType!"
     )
 
-    set __ExtraCmakeArgs=!__ExtraCmakeArgs! %__CMakeClrBuildSubsetArgs% "-DCLR_CROSS_COMPONENTS_BUILD=1" "-DCLR_CMAKE_TARGET_ARCH=%__BuildArch%" "-DCLR_CMAKE_TARGET_OS=%__TargetOS%" "-DCLR_CMAKE_PGO_INSTRUMENT=0" "-DCLR_CMAKE_OPTDATA_PATH=%__PgoOptDataPath%" "-DCLR_CMAKE_PGO_OPTIMIZE=0" %__CMakeArgs%
+    set __ExtraCmakeArgs=!__ExtraCmakeArgs! "-DCLR_CROSS_COMPONENTS_BUILD=1" "-DCLR_CMAKE_TARGET_ARCH=%__BuildArch%" "-DCLR_CMAKE_TARGET_OS=%__TargetOS%" "-DCLR_CMAKE_PGO_INSTRUMENT=0" "-DCLR_CMAKE_OPTDATA_PATH=%__PgoOptDataPath%" "-DCLR_CMAKE_PGO_OPTIMIZE=0" %__CMakeArgs%
```

Comment:
Please double-check what cross-components we are building now.

Reply:
For windows arm64, these are the files that we build (as expected):

>  Directory: D:\source\runtime2\artifacts\bin\coreclr\windows.arm64.Debug\x64
> Mode                 LastWriteTime         Length Name
> ----                 -------------         ------ ----
> d----            4/8/2021  5:05 PM                dactabletools
> d----            4/8/2021  5:05 PM                PDB
> -a---            4/8/2021  4:54 PM        9082368 clrjit.dll
> -a---            4/8/2021  4:55 PM       10795520 crossgen.exe
> -a---            4/8/2021  4:54 PM        1260544 jitinterface_x64.dll
> -a---            4/8/2021  4:55 PM        6990336 mscordaccore.dll
> -a---            4/8/2021  4:55 PM        7041024 mscordbi.dll

Reply:
And the PDB folder has PDBs for each of the dlls and crossgen.

Reply:
Thank you!